backends/tfhe-cuda-backend/build.rs

```diff
@@ -62,6 +62,7 @@ fn main() {
             "cuda/include/integer/integer.h",
             "cuda/include/keyswitch.h",
             "cuda/include/linear_algebra.h",
+            "cuda/include/pbs/fft.h",
```

Comment:
The bindings.rs file should get updated in this PR, doesn't it get updated automatically when you build?